test/lit/passes/O1_skip_passes.wast

```diff
@@ -0,0 +1,71 @@
+;; RUN: foreach %s %t wasm-opt -O2 --skip-pass=coalesce-locals --skip-pass=simplify-locals --skip-pass=simplify-locals-nostructure -S -o - 2>&1 | filecheck %s
+
+;; Check that we can skip several passes. Note that no local.tee is introduced.
+
+;; There should also be no warning in the output.
+;; CHECK-NOT: warning:
+
+(module
+ ;; CHECK:      (type $0 (func (param i32 i32)))
+
+ ;; CHECK:      (type $1 (func (param i32)))
+
+ ;; CHECK:      (import "a" "b" (func $log (param i32 i32)))
+ (import "a" "b" (func $log (param i32 i32)))
+
+ (func $foo (export "foo") (param $p i32)
+  ;; The locals $x and $y can be coalesced into a single local, but as we do not
+  ;; run that pass, they will not be. Other minor optimizations will occur here,
+  ;; such as using a tee.
```

Comment:
This comment says a tee will happen, which contradicts the previous one if I am not mistaken?

Reply:
I copied the test from `O1_skip.wast` but forgot to update all the comments. This is fixed.